linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/AccessLoggerTest.scala

```diff
@@ -1,15 +1,15 @@
 package io.buoyant.linkerd.protocol.http
 
+import java.util.{TimeZone, logging => javalog}
+
```

Comment:
Tiny style nit: Linkerd style is to group all imports together (no blank line inbetween) in alphabetical order.